qiskit/ignis/mitigation/expval/base_meas_mitigator.py

```diff
@@ -166,13 +166,13 @@ def stddev_upper_bound(self, shots: int = 1, qubits: Optional[List[int]] = None)
         return gamma / np.sqrt(shots)
 
     def plot_assignment_matrix(self,
-                               qubits: Optional[List[int]] = None,
-                               ax: Optional[plt.axes] = None) -> plt.axes:
+                               qubits=None,
+                               ax=None):
         """Matrix plot of the readout error assignment matrix.
 
         Args:
-            qubits: Optional, qubits being measured for operator expval.
-            ax: Optional. Axes object to add plot to.
+            qubits (list): Optional, qubits being measured for operator expval.
```

Comment:
I'd recommend using ``list(int)`` here (also, in ``plot_mitigation_function()``), so it's clear that you're not referring to some QRegister or other type of items.

Reply:
Sure, done in: https://github.com/Qiskit/qiskit-ignis/pull/533/commits/4d2be75cd61cfd2942bb75b13ba91e9a4312c757